index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio skeleton</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <main>
+
+    <div class="container"></div>
+      <header>
+        <nav class="navbar">
+          <div class="header-top-art">
+            <img src="./images/image_geometry_1.png" alt="top left art"/>
+          </div>
+       
+          <div class="menu-ham">
+            <img src="/images/icon-menu.svg" alt="ham icon menu"/>
+          </div>
+        </nav>
+      </header>
+
+
+
+      <section class="intro">
+
+        <div>
+          <h1 class="name">Charlie Giorgania</h1>
+          <p class="intro-text">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+          </p>
+        </div>
+        <div class="start-collab-box">
+          <div class="start-collab-btn">
+            <a href="#" class="start-content">Start collaboration</a>
+          </div>
```

Comment:
- [ ] You should center align the `start collaboration` button horizontally as in the Figma design. 
![figma-start-collab-button-and-decor-img](https://user-images.githubusercontent.com/44057863/164609446-0aa30b14-93d0-450f-ab11-9501496583f4.png)
![andym-start-collab-btn-and-decor-img](https://user-images.githubusercontent.com/44057863/164609461-c8996618-1af5-483f-ae9c-884117ed1873.png)